albumentations/augmentations/transforms.py

```diff
@@ -3496,3 +3498,80 @@ def get_transform_init_args_names(self):
             "beta_limit",
             "noise_limit",
         )
+
+
+class PixelDropout(ImageOnlyTransform):
+    """Set pixels to 0 with some probability.
+
+    Args:
+        dropout_prob (float): pixel drop probability. Default: 0.01
+        per_channel (bool): if set to `True` drop mask will be sampled fo each channel,
+            otherwise the same mask will be sampled for all channels. Default: False
+        drop_value (number or sequence of numbers or None): Value that will be set in dropped place.
+            If set to None value will be sampled randomly, default rangeges will be used:
+                - uint8 - [0-255]
+                - int32 - [INT_MIN, INT_MAX]
+                - float, double - [0, 1]
+            Default: 0
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+    Image types:
+        any
+    """
+
+    def __init__(
+        self,
+        dropout_prob: float = 0.01,
+        per_channel: bool = False,
+        drop_value: typing.Optional[typing.Union[float, typing.Sequence[float]]] = 0,
+        always_apply: bool = True,
```

Comment:
Is it a reason to set `always_apply` to `True` by default?

Reply:
oops, it is typo